internal/service/http/http.go

```diff
@@ -120,7 +122,8 @@ func New(opts Options) *Service {
 		tcpLis:    tcpLis,
 		memLis:    memconn.NewListener(l),
 
-		componentHttpPathPrefix: "/api/v0/component/",
+		componentHttpPathPrefix:          "/api/v0/component/",
+		componentHttpPathPrefixRemotecfg: "/api/v0/component/remotecfg",
```

Comment:
Where is `remotecfg` endpoint being called from?

Reply:
Presently, this is used so that lookups via the in-memory listener work properly. There isn't any other caller for now.

Foe example, currently, if you try and scrape `prometheus.exporter.self` for example on a remotecfg-provided config, it would fail with an 'invalid URL' message.